cmd/broker/main_suite_test.go

```diff
@@ -36,6 +41,37 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	bytes, err := json.Marshal([]string{binPath})
 	Expect(err).NotTo(HaveOccurred())
 
+	cli, err = dockerclient.NewEnvClient()
+	Expect(err).NotTo(HaveOccurred())
+
+	// Deleting this image so that integration tests will fail if broker never pulls the image down.
+	// The image being used is shown in eth.json as `nimak/eth-dev`. We only delete latest so other tagged
+	// images are unaffected
```

Comment:
>  We only delete latest so other tagged images are unaffected

So what?

Reply:
trying to minimize the amount of stuff we delete on a person's machine.

Reply:
can un-tag instead of delete, depending on how pushed, there may not be a latest. There is no functional difference between deleting `nimak/eth-dev` and `nimak/eth-dev:latest` that I can recall.